Basim_Muhammad/turtles/apps/turtles_app/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render,redirect
+
+def index(request):
+	return render (request,'index.html')
+
+def show_pic(request,color):
+	if color == 'blue':
+		return render(request,'blue.html')
+	elif color == 'orange':
+		return render(request,'orange.html')
+	elif color == 'red':
+		return render(request,'red.html')
+	elif color == 'purple':
+		return render(request,'purple.html')
+	elif color=='':
+		return render(request,'tmnt.html')
+	else:
+		return render(request,'worng.html')
+	
```

Comment:
Instead of creating a separate html file for each picture, you can just create one and when you render it just send it the name of the image file you want to use based on the route params.